cli/args.go

```diff
@@ -69,6 +71,7 @@ func parseTerragruntOptionsFromArgs(args []string) (*options.TerragruntOptions,
 		Logger: util.CreateLogger(""),
 		RunTerragrunt: runTerragrunt,
 		Source: terraformSource,
+		UpdateSource: updateSource,
```

Comment:
Is the preferred phrasing "updateSource" or "sourceUpdate"? It seems to be used in both ways in different places.